### PR TITLE
Improve coverage in `src/api/subscribe-response`

### PR DESCRIPTION
**Pull request by ChrisAntaki**
- Fully covers a partially covered line
image

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
That makes sense, I'll add more tests

**Comment by ChrisAntaki** [comment]
Tests are added now, and this is ready for another review

File: src/api/subscribe-response-test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      expect(mockCompleteHandler).not.to.be.called;
+      subscribeResponse.complete();
+      expect(mockCompleteHandler).to.be.called;
+    });
+  });
+});
+
+describes.realWin('PurchaseData', () => {
+  describe('json', () => {
+    it('returns JSON', () => {
```
Comment (fikus) [inline src/api/subscribe-response-test.js:90]: nit: It does not return JSON (which would be a string). It returns an object.

**Review by fikus — APPROVED** [review]

File: src/api/subscribe-response-test.js (hunk 1 above)
Comment (ChrisAntaki) [reply src/api/subscribe-response-test.js]: Nice catch, I updated the description to "returns a serializable object"

**Pull request merged**